src/aelfrice/cli.py

```
@@ -737,6 +743,251 @@ def _cmd_search(args: argparse.Namespace, out: object) -> int:
+                for combined, h, action, relatedness in candidates
+            ],
+        }
+        print(json.dumps(payload2, indent=2), file=out)  # type: ignore[arg-type]
+        return 0
+
+    print(f"seed: {seed_b.id}: {seed_b.content}", file=out)  # type: ignore[arg-type]
+    if not candidates:
+        print("(no candidates — seed has no outbound edges)", file=out)  # type: ignore[arg-type]
+        return 0
+    print(f"top {len(candidates)} consolidation candidate(s):", file=out)  # type: ignore[arg-type]
+    for combined, h, action, relatedness in candidates:
```

Ruff (0.15.12)

[warning] Loop control variable `relatedness` not used within loop body

(B007)